contracts/jobs/JobsManager.sol

```diff
@@ -1,19 +1,30 @@
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.13;
 
 import "./IJobsManager.sol";
-import "../Controllable.sol";
-import "../LivepeerProtocol.sol";
+import "./libraries/JobLib.sol";
+import "./libraries/MerkleProof.sol";
+import "../Manager.sol";
+import "../ContractRegistry.sol";
+import "../LivepeerToken.sol";
 import "../bonding/IBondingManager.sol";
 import "../verification/Verifiable.sol";
 import "../verification/Verifier.sol";
-import "./libraries/TranscodeJobs.sol";
 
-contract JobsManager is IJobsManager, Verifiable, Controllable {
-    using TranscodeJobs for TranscodeJobs.Jobs;
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ECRecovery.sol";
+
+contract JobsManager is IJobsManager, Verifiable, Manager {
+    using SafeMath for uint256;
+
+    // Token address
+    LivepeerToken public token;
 
     // Verifier address
     Verifier public verifier;
 
+    // Content-addressed storage hash of verification code
+    string public verificationCodeHash;
```

Comment:
This should be removed from `JobsManager` because `verificationCodeHash` is now set in `OraclizeVerifier`. This was prompted because I was running into a `Stack too deep` compiler error for `JobsManager.verify()` which passed in `verificationCodeHash` as a parameter to `verifier.verify()` (https://github.com/livepeer/protocol/blob/71698abc2887a596c63745d02851513ec26bd62e/contracts/jobs/JobsManager.sol#L280). The compiler error was solved by removing `verificationCodeHash` as a parameter to `verifier.verify()` and instead having it be set in `OraclizeVerifier` which is currently the only contract that uses it